pyproject.toml

```diff
@@ -0,0 +1,70 @@
+[build-system]
+requires = [ "setuptools>=61", "wheel", "setuptools-git-versioning>=2.0,<3", ]
+build-backend = "setuptools.build_meta"
+
+[tool.setuptools-git-versioning]
+enabled = true # Allows setuptools-git-versioning to pull version from git state
+
+[tool.setuptools]
+py-modules = []
```

Comment:
B: Won't this trigger the issue of no exported modules? Rely on autodiscovery instead

Reply:
I think we do need something as otherwise e.g. the examples folder causes issues, I've updated to copy tubulars setup https://github.com/lvgig/model_interpreter/pull/2/commits

for reference https://github.com/lvgig/tubular/blob/main/pyproject.toml

Reply:
I tried to build the package - wowza that sent me off on a goose chase!

`setup.py` makes hard reference to `requirements.txt`, so we were never going to be able to release these changes. So I tweaked what needed to change on the branch package-structure and opened PR #7 with a working config and release pipeline. Either merge it in, or copy the changes that you feel are worthwhile.

I think that your change to py-modules fixed my comment, but we can't release it without fixing the other stuff.